src/installer/pkg/sfx/Microsoft.NETCore.App/PackageOverrides.txt

```diff
@@ -94,36 +100,52 @@ System.IO.Compression|4.3.0
 System.IO.Compression.ZipFile|4.3.0
 System.IO.FileSystem|4.3.0
 System.IO.FileSystem.AccessControl|4.4.0
-System.IO.FileSystem.DriveInfo|4.3.0
+System.IO.FileSystem.DriveInfo|4.3.1
 System.IO.FileSystem.Primitives|4.3.0
 System.IO.FileSystem.Watcher|4.3.0
 System.IO.IsolatedStorage|4.3.0
 System.IO.MemoryMappedFiles|4.3.0
 System.IO.Pipes|4.3.0
+System.IO.Pipes.AccessControl|5.0.0
+System.IO.Pipelines|9.0.0
 System.IO.UnmanagedMemoryStream|4.3.0
 System.Linq|4.3.0
 System.Linq.Expressions|4.3.0
+System.Linq.Parallel|4.3.0
 System.Linq.Queryable|4.3.0
-System.Net.Http|4.3.0
+System.Memory|5.0.0
+System.Net.Http|4.3.4
+System.Net.Http.Json|9.0.0
 System.Net.NameResolution|4.3.0
-System.Net.Primitives|4.3.0
+System.Net.NetworkInformation|4.3.0
+System.Net.Ping|4.3.0
+System.Net.Primitives|4.3.1
 System.Net.Requests|4.3.0
-System.Net.Security|4.3.0
+System.Net.Security|4.3.2
 System.Net.Sockets|4.3.0
 System.Net.WebHeaderCollection|4.3.0
+System.Net.WebSockets|4.3.0
+System.Net.WebSockets.Client|4.3.2
+System.Numerics.Vectors|5.0.0
 System.ObjectModel|4.3.0
 System.Private.DataContractSerialization|4.3.0
+System.Private.Uri|4.3.2
 System.Reflection|4.3.0
-System.Reflection.Emit|4.3.0
-System.Reflection.Emit.ILGeneration|4.3.0
-System.Reflection.Emit.Lightweight|4.3.0
+System.Reflection.DispatchProxy|6.0.0
```

Comment:
I think this one can stay at `5.0.0`, we never shipped a `5.0.0` (just 5.0.0-preview packages for a while), and it makes it consistent with the other 4.x packages.